charts/cp-schema-registry/templates/pdb.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
 spec:
```

Comment:
In the same PR, you should rename the PDB object (cf line just above) and remove the `-pdb` suffix.

Reply:
👍